scroll-animations-1/Overview.bs

```diff
@@ -589,42 +615,178 @@ Computed value: As specified
+'animation-name' has the following effects:
 
-    The <<time>> value, if specified, determines the timeline's {{timeRange}}.
+:   none
+::  No timelines and keyframes are specified at all, so there will be no
+    animation. Any other animations properties specified for this animation have no
+    effect.
+
+
+:   <<keyframes-name>>
+::  If ''@timeline'' rule with  the name specified by <<keyframes-name>>, then
+    the animation's [=timeline=] is a timeline whose property values are
+    provided by that rule. Otherwise the animation's [=timeline=] is a
+    {{DocumentTimeline}}, more specifically the <a>default document
+    timeline</a>.
+
+Note: Notice that the behavior for the case when no timeline with given name was
+found is different for either property. This ensures backward compatibility
+since most existing animations which have 'animation-name' specified expect to
+use <a>default document timeline</a>.
+
+
+## The '@timeline' at-rule ## {#timeline-at-rule}
+
+[=Scroll Timelines=] are specified in CSS using the <dfn>@timeline</dfn>
+at-rule, defined as follows:
+
+<pre>
+  @timeline = @timeline <<timeline-name>> { <<declaration-list>> }
+
+
+  <dfn>&lt;timeline-name></dfn> = <<custom-ident>> | <<string>>
+</pre>
+
+
+An ''@timeline'' rule has a name given by the <<custom-ident>> or <<string>> in
+its prelude. The two syntaxes are equivalent in functionality; the name is the
+value of the ident or string. As normal for <<custom-ident>>s and <<string>>s,
+the names are fully case-sensitive; two names are equal only if they are
+codepoint-by-codepoint equal. The <<custom-ident>> additionally excludes the
+none keyword.
+
+
+The <<declaration-list>> inside of ''@timeline'' rule can only contain the
+descriptors defined in this section.
+
+### the 'timeline-type' descriptor ### {#timeline-type-descriptors}
+
+
+<pre class='descdef'>
+  Name: timeline-type
+  For: @timeline
+  Value: scroll | time
+  Initial: time
+</pre>
 
-    The <<single-animation-fill-mode>> value, if specified, determines the timeline's {{fill}}.
+The 'timeline-type' determines the type of timeline and whose possible values
+have the following effects:
+
+:   time
+::  The animation's [=timeline=] is a {{DocumentTimeline}}, more specifically
+    the <a>default document timeline</a>.
+
+:   scroll
+::  The animation's [=timeline=] is a {{ScrollTimeline}} as specified in the
+    ''@timeline'' rule's <<declaration-list>>.
+
+
+
+### Scroll Timeline descriptors ### {#scroll-timeline-descriptors}
+
+
+<pre class='descdef'>
+  Name: scroll-source
+  For: @timeline
+  Value: selector(<<id-selector>>) | none
+  Initial: none
+</pre>
+
+'scroll-source' descriptor determines the scroll timeline's {{source}}. If
+specified as an '<selector()>' the scroll timeline's {{source}} is the
+[=scroll container=] identified by the <<id-selector>>, otherwise if not
+specified or none then it is the animation target's nearest scrollable ancestor.
```

Comment:
Based on [this](https://drafts.csswg.org/scroll-animations/#scrolltimeline-interface ) ScrollTimeline.source attribute is set to document.scrollingElement if no scroll source is passed in the constructor. I would expect the same behavior for javascript and CSS specification of scroll timeline.

Reply:
Good point. There was a note in CSSWG group discussion to use nearest ancestor. But you are right
about consistency. For now I changed it to use scrollingElement same as Javascript.

Added an issue note to consider using nearest ancestor in this case. We can revisit this in future.